examples/java/src/main/java/org/apache/beam/examples/cookbook/BigQueryTornadoes.java

```diff
@@ -179,14 +180,16 @@ static void runBigQueryTornadoes(Options options) {
                   BigQueryIO.readTableRows()
                       .fromQuery(options.getInputQuery())
                       .usingStandardSql()
-                      .withMethod(Method.DIRECT_READ));
+                      .withMethod(Method.DIRECT_READ)
+                      .withFormat(DataFormat.AVRO));
         } else {
           rowsFromBigQuery =
               p.apply(
                   BigQueryIO.readTableRows()
                       .from(options.getInput())
                       .withMethod(Method.DIRECT_READ)
-                      .withSelectedFields(Lists.newArrayList("month", "tornado")));
+                      .withSelectedFields(Lists.newArrayList("month", "tornado"))
+                      .withFormat(DataFormat.AVRO));
```

Comment:
Hm. So `BigQueryIO.readTableRows()` will fail now if `.withFormat()` isn't specified?
This is a breaking change to the API. I think instead we should just default to use AVRO when format isn't specified.

Reply:
Yes, by the recent change on throwing IllegalArgument when format is unspecified. I will change the default to AVRO then. The explicit `.withFormat()` to the test should be kept anyway? or also removed?

Reply:
Ah, sorry, I didn't realize that https://github.com/apache/beam/pull/14586#discussion_r659087861 was the only place where you were handling the default, normally we specify defaults in the IO creator methods (see comment above).

Please remove the `.withFormat(DataFormat.AVRO)`.